.homework.js

```diff
@@ -0,0 +1,92 @@
+//1 
+
+const summation =[1, 2, 3, 4, 5]
+summation.reduce((a,b) => a + b, 0)
+
+//2
+const summationEven = function (){
+  let arr =[1, 2, 3, 4, 5];
+for (let i =0; i<= 20; i++) 
+{
+  if (i % 2 ==0)
+    console.log(`${i} Even number`);
+   else
+     console.log(`${i} Odd number`);
+  }
+ 
+}
+
+//3 
+
+let avg = arr => arr.reduce( ( p, c ) => p + c, 0 ) / arr.length;
+avg( [ 8, 2, 2, 4] );
+
+//4*
+let revWord = "caterpillar"
+word = revWord.reverse();
```

Comment:
`revWord` is a string and `.reverse()` is an array method it doesn't work with strings.
Think of a way you can use this method, consider string methods like `.split()` to turn your string into an array.